airbyte-persistence/job-persistence/src/test/java/io/airbyte/persistence/job/DefaultJobPersistenceTest.java

```diff
@@ -285,13 +285,7 @@ void testWriteOutput() throws IOException, SQLException {
     assertEquals(Optional.of(jobOutput), updated.getAttempts().get(0).getOutput());
     assertNotEquals(created.getAttempts().get(0).getUpdatedAtInSecond(), updated.getAttempts().get(0).getUpdatedAtInSecond());
 
-    final Optional<Record> record =
```

Comment:
Replacing jobId and attemptNumber also simplifies how the methods are used. 

The alternative is exposing a helper method to calculate the `attemptId` - this feels unnecessary.